src/main/scala/zio/prelude/Validation.scala

```diff
@@ -34,17 +34,6 @@ sealed trait Validation[+E, +A] { self =>
   final def <&>[E1 >: E, B](that: Validation[E1, B]): Validation[E1, (A, B)] =
     zipPar(that)
 
-  /**
-   * Returns whether this `Validation` and the specified `Validation` are equal
-   * to each other.
-   */
-  override final def equals(that: Any): Boolean =
```

Comment:
We definitely need to keep this because we need to maintain the invariant that equality for `Validation` values is independent of the other of errors.

Reply:
We could potentially do that. At the time we implemented this we didn't have that data structure and it isn't necessary here. We can just use a `NonEmptyChunk` and ignore its ordering. This can be more efficient if we don't need the other properties of `Set` like efficient lookup but just basically need a collection that can be iterated over with an efficient append and concatenation operation.